File: ObjetoScala/Persona.scala
package ObjetoScala

class Persona(
                val strNombre: String,
                val strApellidos: String,
                val intEdad: Int){


  def Imprimir(strNombre: String):Unit= {
    println(s"Mi nombre es $strNombre, " +
      s"y mi apellido es $strApellidos, " +
      s"con edad de $intEdad años")
  }
  Imprimir(strNombre)

  override def toString = s"Persona($strNombre, $strApellidos, $intEdad)"


}

File: ObjetosJava/Demo/Persona.java
package ObjetosJava.Demo;

public class Persona {
    private String nombre ;
    private String apellido ;
    private String edad ;

    public Persona(String nombre, String apellido, String edad) {
        this.nombre = nombre;
        this.apellido = apellido;
        this.edad = edad;
    }

    public Persona() {

    }


    @Override
    public String toString() {
        return "Persona{" +
                "nombre='" + nombre + '\'' +
                ", apellido='" + apellido + '\'' +
                ", edad='" + edad + '\'' +
                '}';
    }
}
